iceoryx_hoofs/include/iceoryx_hoofs/log/building_blocks/console_logger.hpp

```diff
@@ -64,35 +70,39 @@ class ConsoleLogger
 
     void assumeFlushed() noexcept;
 
+    // AXIVION Next Construct AutosarC++19_03-A3.9.1 : Not used as an integer but a low-level C-style string
     void logString(const char* message) noexcept;
```

Comment:
Could we not use a `string_view` here? Those kind of interfaces always give me a bit of a shiver as the size is not passed along :grimacing: Would be something that could be solved after #1723 is implemented?

Reply:
Well, this is explicitly meat to log a C-string e.g. `IOX_LOG(INFO) << "C-string"`. When this is wrapped it is not possible anymore. At some places we have to deal with this low level constructs.

Reply:
@elBoberido I know, still feeling like :grimacing: `IOX_LOG(INFO) << string_view("C-string")` would be overt the top. Let's leave it like this.